examples/gocean/eg1/README.md

```diff
@@ -16,6 +16,17 @@ python ./runme.py
 
 Examine the runme*.py scripts themselves for further details.
 
+
+## OpenCL PSyclone script
+
+The OpenCL transformation is provided with a PSyclone transformation script
+(`opencl_transformation.py`). This can be run using the PSyclone command:
+
+```sh
+psyclone -s ./opencl_transformation.py -api gocean1.0 shallow_alg.f90
```

Comment:
Do you need the `--kernel-renaming` flag here?

Reply:
We do if we were to compile the generated OpenCL. I added a comment in the Makefile explaining why we need it.